docs/_private_integrations/share-your-private-integration.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Share your private integration to users
+order: 3
+layout: post-toc
+redirect_from: /private_integrations/
+---
+# Share your private integration to users
+
+After you’ve built your private integration, you can invite users to start using your integration in their Zaps. There are two ways you can share your private integration with users:
+
+## Invite users with a public link
+
+You can share a link with users to all versions of your integration. 
+
+> Note:
+> * You can’t view invited users via a public link.
+> * You can’t revoke a link invite, once shared. If you require control over who accesses your private integration, we recommend inviting users by email. 
+
+
+<ol> 
+ <li><a href="https://zapier.com/app/developer">Log into the Zapier Platform UI</a> or refer to the <a href="https://platform.zapier.com/cli_tutorials/getting-started#invite-users-to-your-app">Zapier Platform CLI documentation</a>.</li>
+ <li>Select <b>My Integration</b>.</li>
+ <li>In the <i>Integrations</i> section, select your <b>integration</b>.</li>
+ <li>In the left sidebar, under <i>Manage</i> click <b>Sharing</b>.</li>
+ <li>Click <img style="vertical-align: middle;" src="https://res.cloudinary.com/zapier-media/image/upload/zinnia-icons/actionCopy.svg" alt="ICON NAME icon" width="24"> to copy the invite public link.</li>
+</ol>
```

Comment:
Here's the all-markdown alternative from my [previous comment](https://github.com/zapier/visual-builder/pull/439/files#r1163039391):
```suggestion
1. [Log into the Zapier Platform UI](https://zapier.com/app/developer) or refer to the [Zapier Platform CLI documentation](https://platform.zapier.com/cli_tutorials/getting-started#invite-users-to-your-app).
1. Select **My Integration**.
1. In the *Integrations* section, select your **integration**.
1. In the left sidebar, under *Manage* click **Sharing**.
1. Click <img style="vertical-align: middle; width: 24px;" src="https://res.cloudinary.com/zapier-media/image/upload/zinnia-icons/actionCopy.svg" alt="ICON NAME icon"> to copy the invite public link.
```

Reply:
I've updated in my recent commit to Markdown! Thanks for showing me.